scripts/rv_expression_association/prepare_inputs.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python3
+
+import click
+import logging
+import re
+import subprocess
+import sys
+import pandas as pd
+import xarray as xr
+from cloudpathlib import AnyPath
+from cpg_utils import to_path
+from cpg_utils.hail_batch import output_path
+
+subprocess.run([sys.executable, '-m', 'pip', 'install', 'limix==3.0.4'], check=True)
+subprocess.run(
+    [sys.executable, '-m', 'pip', 'install', 'pandas_plink==2.2.9'], check=True
+)
+
+from pandas_plink import read_plink1_bin  # lint complains about these but surely i need to install them before i can import them? 
+from limix.qc import quantile_gaussianize
+
+# use logging to print statements, display at info level
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
+
+
+@click.command()
+@click.option('--cell-type', required=True, help='More info here')
+@click.option('--gene-name', required=True)  # 'VPREB3'
+@click.option(
+    '--sample-mapping-file', required=True
+)  # 'scrna-seq/grch38_association_files/OneK1K_CPG_IDs.tsv'
+@click.option(
+    '--genotype-file-bed', required=True
+)  # 'v0/plink_files/vpreb3_rare_promoter.bed'
+@click.option(
+    '--genotype-file-bim', required=True
+)  # 'v0/plink_files/vpreb3_rare_promoter.bim'
+@click.option(
+    '--genotype-file-fam', required=True
+)  # 'v0/plink_files/vpreb3_rare_promoter.fam'
+@click.option(
+    '--phenotype-file', required=True
+)  # 'scrna-seq/grch38_association_files/expression_files/B_naive_expression.tsv'
+@click.option('--kinship-file', required=True)  # 'v0/skat/grm_wide.csv'
+@click.option(
+    '--output-folder', required=False, default=''
+)  # by default current directory, where you are running your script from
+def main(
+    cell_type: str,
+    gene_name: str,
+    sample_mapping_file: str,
+    genotype_file_bed: str,
+    genotype_file_bim: str,
+    genotype_file_fam: str,
+    phenotype_file: str,
+    kinship_file: str,
+    output_folder: str,
+):
+
+    expression_filename = AnyPath(output_path(f'{gene_name}_{cell_type}.csv'))
+    genotype_filename = AnyPath(output_path(f'{gene_name}_rare_regulatory.csv'))
+    kinship_filename = AnyPath(output_path('kinship_common_samples.csv'))
+
+    ##################### lint also not happy with these, should I care?
```

Comment:
What do you mean by this? What's the linter complaining about?

Reply:
"too many leading '#' for block comment"

Reply:
Maybe just switch to regions in case you like those!